cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/RecoverFromStorageCommand.java

```diff
@@ -657,4 +652,27 @@ public void close() {
         }
     }
 
+    private void validateArguments() {
+        Preconditions.checkArgument(getArgCount() >= 2, "Incorrect argument count.");
+        final String container = getArg(1);
+
+        if (!NumberUtils.isNumber(container)) {
+            Preconditions.checkArgument(container.toLowerCase().equals("all"), "Container argument should either be ALL/all or a container id.");
```

Comment:
maybe we should update the descriptor to reflect this new change.

Reply:
Added